CWP/papers/HSF-CWP-2017-01_roadmap/latex/cwp-main.tex

```diff
@@ -352,7 +352,7 @@ \section{Software and Computing
 Xeon architecture. More extreme is an architecture that would see
 specialised processing units dedicated to particular tasks, but with
 possibly large parts of the device switched off most of the time,
-so-called dark silicon.
+so-called dark silicon~\cite{IntelCSA}.
```

Comment:
@graeme-a-stewart also added a citation at this place, albeit a different one. Would it make sense to have both? I confess I did not compare the content both links point to.

Reply:
I think so... There will probably be a conflict when of the PR is merged but it should be easy to fix.

Reply:
OK, great. Thanks.

Reply:
Hmmm, I am not sure we should cite something that doesn't exist yet, except in Intel's head. In my PR I added a much more robust citation to dark silicon in general.

Reply:
I saw your citation and I think it is great. I have no problem to remove mine, I added it because @davidlange6 commented that it was an interesting reference. I would not think it is necessarily a bad reference to illustrate potentially more disruptive architectures to use something about the ideas being investigated by big players...